### PR TITLE
Readme updates for env vars

### PR DESCRIPTION
**Pull request by bpathak-ons**
### What

* Readme updates with env vars used in application

### How to review

* Look at Configuration class and match the new entries in the document

### Who can review

* anyone not me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,11 +109,36 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
...
+| AUDIT_DB_ENABLED_ENV_VAR                          | "audit_db_enabled"                          | Audit DB
+| MATHJAX_SERVICE_URL                               | "http://localhost:8888"                     | Mathjax service URL
+| DATASET_API_URL                                   | "http://localhost:22000"                    | Dataset API URL
+| IMAGE_API_URL                                     | "http://localhost:24700"                    | Image API URL
+| KAFKA_ADDR                                        | "localhost:9092"                            | The address of Kafka
+| KAFKA_CONTENT_PUBLISHED_TOPIC                     | "content-published";                        | Topic for endpoint /publisheddata
+| ENABLE_KAFKA                                      | false                                       | Feature flag to send kafka messages when a collection is published
+| KAFKA_ADDR                                        | localhost:9092                              | Comma seperated list of kafka brokers
+| KAFKA_CONTENT_PUBLISHED_TOPIC                     | content-published                           | Kafka topic to send content pubished event to
```
Comment (outsidecoder) [inline README.md:132]: Two entries for the same Env Variables - KAFKA_CONTENT_PUBLISHED_TOPIC

File: README.md [hunk 2]
```diff
@@ -109,11 +109,36 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
...
+| DEFAULT_FLORENCE_URL                              | "http://localhost:8081"                     | Florence URL
+| DEFAULT_BRIAN_URL                                 | "http://localhost:8083"                     | Brian URL
+| DEFAULT_TRAIN_URL                                 | "http://localhost:8084"                     | Train URL
+| DEFAULT_DYLAN_URL                                 | "http://localhost:8085"                     | Dylan URL
+| CONTENT_DIRECTORY                                 | "content"                                   | Content directory
+| AUDIT_DB_ENABLED_ENV_VAR                          | "audit_db_enabled"                          | Audit DB
+| MATHJAX_SERVICE_URL                               | "http://localhost:8888"                     | Mathjax service URL
+| DATASET_API_URL                                   | "http://localhost:22000"                    | Dataset API URL
+| IMAGE_API_URL                                     | "http://localhost:24700"                    | Image API URL
+| KAFKA_ADDR                                        | "localhost:9092"                            | The address of Kafka
```
Comment (outsidecoder) [inline README.md:128]: Two of the same Env Variables in table.  Also note one has quotes, one does not.

**Review by outsidecoder — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -109,13 +109,36 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
...
+| DEFAULT_SLACK_USERNAME                            | "Zebedee"                                   | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                        | "http://localhost:8080"                     | Service public URL
+| DEFAULT_FLORENCE_URL                              | "http://localhost:8081"                     | Florence URL
+| DEFAULT_BRIAN_URL                                 | "http://localhost:8083"                     | Brian URL
+| DEFAULT_TRAIN_URL                                 | "http://localhost:8084"                     | Train URL
+| DEFAULT_DYLAN_URL                                 | "http://localhost:8085"                     | Dylan URL
+| CONTENT_DIRECTORY                                 | "content"                                   | Content directory
+| KAFKA_SEC_PROTO                                   | _unset_                                     | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_KEY_P12                          | _unset_                                     | if using TLS (see above), this is a base64-encoded PKCS12 keystore (optional, used for client auth)
```
Comment (DavidSubiros) [inline README.md:125]: I didn't see this var being used in the code, where is it used?
Instead, I saw the following vars being used if kafka tls is enabled:
- KAFKA_SEC_CLIENT_KEY
- KAFKA_SEC_CLIENT_CERT

I think `KAFKA_SEC_CLIENT_KEY_P12` should be removed and  `KAFKA_SEC_CLIENT_KEY` and `KAFKA_SEC_CLIENT_CERT` should be added, unless I am missing something

**Review by DavidSubiros — COMMENTED** [review]
LGTM - but identified a mismatch with kafka tls env vars

**Review by DavidSubiros — APPROVED** [review]
LGTM

**Pull request merged**